Lib/test/test_ctypes/test_parameters.py

```diff
@@ -240,7 +240,6 @@ def test_parameter_repr(self):
         self.assertRegex(repr(c_ulonglong.from_param(20000)), r"^<cparam '[LIQ]' \(20000\)>$")
         self.assertEqual(repr(c_float.from_param(1.5)), "<cparam 'f' (1.5)>")
         self.assertEqual(repr(c_double.from_param(1.5)), "<cparam 'd' (1.5)>")
-        self.assertEqual(repr(c_double.from_param(1e300)), "<cparam 'd' (1e+300)>")
```

Comment:
Please don't remove tests. Maybe skip it if `if getattr(sys, 'float_repr_style', '') == 'short':`.